modules/core/04-channel/types/msgs.go

```diff
@@ -499,18 +499,57 @@ var _ sdk.Msg = &MsgChannelUpgradeInit{}
 
 // NewMsgChannelUpgradeInit constructs a new MsgChannelUpgradeInit
 // nolint:interfacer
-func NewMsgChannelUpgradeInit() *MsgChannelUpgradeInit {
-	return &MsgChannelUpgradeInit{}
+func NewMsgChannelUpgradeInit(
+	portID, channelID string,
+	proposedUpgradeChannel Channel,
+	counterpartyTimeoutHeight clienttypes.Height,
+	counterpartyTimeoutTimestamp uint64,
+	signer string,
+) *MsgChannelUpgradeInit {
+	return &MsgChannelUpgradeInit{
+		PortId:                       portID,
+		ChannelId:                    channelID,
+		ProposedUpgradeChannel:       proposedUpgradeChannel,
+		CounterpartyTimeoutHeight:    counterpartyTimeoutHeight,
+		CounterpartyTimeoutTimestamp: counterpartyTimeoutTimestamp,
```

Comment:
Should we do `TimeoutHeight`, `TimeoutTimestamp` to keep in line with the packet timeout naming, since the semantics are the same?

cc: @damiannolan

Reply:
Yeah, I'm always happier for shorter naming. Sounds good to me!